### PR TITLE
Toggle enable/disable yast_apparmor also on SLES >12.4

### PR DESCRIPTION
**Pull request by bedi1982**
To correctly load the profiles list yast-apparmor the 'enable' option has to be toggled on/off, this change removes the condition to toggle only on SLES15 and makes it check for all SLES > 12.4

This is a fix for:
https://progress.opensuse.org/issues/56171

Test runs:
15.1 http://deathstar.suse.cz/tests/799
15.0 http://deathstar.suse.cz/tests/800
12.3 http://deathstar.suse.cz/tests/801
12.4 http://deathstar.suse.cz/tests/802

### CODE REVIEW

**Comment by jknphy** [comment]
Hi Sergio, this one breaks sle12sp5: https://openqa.suse.de/tests/3321504#step/yast2_apparmor/6, it seems that the check_screen is needed.

File: tests/console/yast2_apparmor.pm [hunk 1]
```diff
@@ -18,7 +18,7 @@ use utils qw(zypper_call systemctl);
 use version_utils qw(is_pre_15 is_sle is_opensuse is_leap);
 
 sub install_extra_packages_requested {
-    if (check_screen 'yast2_apparmor_extra_packages_requested') {
+    if (assert_screen 'yast2_apparmor_extra_packages_requested') {
```
Comment (rwx788) [inline tests/console/yast2_apparmor.pm]: I believe that if we revert this change, if should fix SLES 12 SP5 tests.
